requirements.txt

```diff
@@ -1,18 +1,18 @@
-dimod==0.12.20
-dwave-optimization==0.4.0
-dwave-preprocessing==0.6.7
-dwave-cloud-client==0.12.0
+dimod==0.12.21
+dwave-optimization==0.6.4
+dwave-preprocessing==0.6.11
+dwave-cloud-client==0.14.1
 dwave-networkx==0.8.14
-dwave-samplers==1.4.0
+dwave-samplers==1.7.0
 homebase==1.0.1
-minorminer==0.2.19
+minorminer==0.2.19      # todo
```

Comment:
Will update this when minorminer is released. Build/tests work with the older version, but they're slower, as minorminer has to be built from source.